api/app/controllers/api/v1/projects_controller.rb

```diff
@@ -37,7 +40,8 @@ def create
 
       def update
         @project = load_and_authorize_project
-        ::Updaters::Project.new(project_params).update(@project)
+        options = @project.draft? || project_params.dig(:draft) ? { touch: false } : {}
```

Comment:
The logic here seems weird. "Any time a request to update a project comes in with a draft param, we don't change the timestamp on the project". What is this trying to accomplish? And why? And if this logic is important, why is it only in this controller?